### PR TITLE
Fixed takeScreenshot method; added saveScreenshot method

### PR DESCRIPTION
**Pull request by Gennadiii**

### CODE REVIEW

**Review by superTes — CHANGES_REQUESTED** [review]
remove ";" symbols at the end of the line

File: lib/client.js [hunk 1]
```diff
@@ -233,6 +244,22 @@ function initializatorClient(requests, opts) {
...
 
+    async saveScreenshot(name, params = {}) {
+      if (!name) {
+        throw new Error(`Name is obligatory to save screenshot`);
+      }
+      const {
+          path = `${process.cwd()}/screenshots`,
+          encoding = 'base64',
+          format = 'png'
+        } = params,
```
Comment (superTes) [inline lib/client.js]: add const for every variable 
const screenshotPath = `${path}/${name}.${format}`
const  screenshot = await this.takeScreenshot()

**Review by superTes — CHANGES_REQUESTED** [review]
need to be fixed

**Review by superTes — CHANGES_REQUESTED** [review]
fix next issues, please

**Comment by Gennadiii** [comment]
@superTes all done

**Review by potapovDim — APPROVED** [review]
👍

**Pull request merged**